binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/internal/YahooWeatherHandlerFactory.java

```diff
@@ -44,4 +48,11 @@ protected ThingHandler createHandler(Thing thing) {
 
         return null;
     }
+
+    @Override
+    protected Map<DefaultPropertyKey, String> getStaticProperties(Thing thing) {
+        Map<DefaultPropertyKey, String> staticProperties = super.getStaticProperties(thing);
+        staticProperties.put(DefaultPropertyKey.VENDOR, "Yahoo");
```

Comment:
A vendor property imho does not make sense for a web service. Nobody "bought" it.